bin/tessel-2.js

```diff
@@ -247,11 +247,6 @@ makeCommand('wifi')
   .help('Configure the wireless connection');
 
 parser.command('key')
-  .option('method', {
-    position: 1,
-    required: true,
-    choices: ['generate'],
```

Comment:
I originally had it this way because I was going to add other choices like `remove`, `reset`, etc. but lost interest. I think this change looks fine.